package.json

```diff
@@ -845,6 +845,14 @@
                     "command": "aws.cwl.viewLogStream",
                     "when": "false"
                 },
+                {
+                    "command": "aws.cwl.changeFilterPattern",
```

Comment:
Is `changeFilterPattern` actually needed at all? Can the search icon just point to `aws.cwl.changeLogSearch` ?

Reply:
I think `aws.cwl.changeFilterPattern` and `aws.cwl. changeTimeFilter` have to be distinct commands in order to allow two distinct icons. There currently is no `aws.cwl.changeLogSearch` command, its just that both `aws.cwl.changeFilterPattern` and `aws.cwl. changeTimeFilter` are registered to the `changeLogSearchParams` function with different params. There may be another way to keep them with distinct icons and function, but this is how I thought to get around that issue.

Reply:
> have to be distinct commands in order to allow two distinct icons.

Yup. This is the best solution I've found so far.

Reply:
> There currently is no aws.cwl.changeLogSearch command

sorry, see updated comment. I copy-pasted the wrong name.

Reply:
Both commands pretty much do the same thing as `aws.cwl.searchLogGroup`, you're just starting from existing parameters. This would be more obvious once you realize that `changeLogSearch.ts` is mostly redundant; we can re-use the wizard implementation.